src/renderer/components/badge/badge.module.css

```diff
@@ -25,6 +25,18 @@
   max-width: 100%;
 }
 
+// might be used in a scrollable area with {overflow: auto}
```

Comment:
Do you mean "must be"?

Reply:
Well, actually not really, because this class is not required for `Badge`.
Also `overflow: auto` is not requirement for parent container and could be different.

Reply:
okay sounds good